pkg/licenses/license.go

```diff
@@ -423,28 +423,183 @@ github.com/aws/aws-sdk-go
    limitations under the License.
 
 ***
-github.com/boltdb/bolt
```

Comment:
I think this change is autogenerated, but do we know why this license is being removed? I think boltdb is still being used

Reply:
BoltDB was removed in https://github.com/aws/amazon-ecs-cni-plugins/pull/113, specifically in https://github.com/aws/amazon-ecs-cni-plugins/pull/113/commits/7ac5bce70c595c782689f28c15a1a0b00601204b when updating to a newer version of `docker/libkv`. `docker/libkv` switched from BoltDB to [BBolt](https://github.com/etcd-io/bbolt) in https://github.com/docker/libkv/pull/202.

I didn't run `make generate` in https://github.com/aws/amazon-ecs-cni-plugins/pull/113 so BoltDB wasn't removed from the license until now. BBolt wasn't added because it's from [go.etcd.io](https://github.com/aws/amazon-ecs-cni-plugins/blob/master/go.mod#L26), and the license script currently only covers github.com and golang.org packages.
https://github.com/aws/amazon-ecs-cni-plugins/blob/c184a8fd874bab7dc7f16bcce4f9e1c6b21013e8/scripts/licenses.sh#L44-L55